linux_container/linux_container.go

```diff
@@ -93,6 +94,8 @@ type LinuxContainer struct {
 	env process.Env
 
 	processIDPool *ProcessIDPool
+
+	link devices.LinkInterface
```

Comment:
rather than having linux_container depend on the whole devices.LinkInterface (most of which it doesn't need), perhaps it could depend just on a `type NetworkStatisticser interface { Statistics() }` defined locally in this package?